_spinnaker/pacrd.md

```diff
@@ -0,0 +1,372 @@
+
+```
+curl -fsSL https://engineering.armory.io/manifests/pacrd-0.1.0.yaml > pacrd-0.1.0.yaml
+```
+
+Then, inspect the manifest to make sure it's compatible with your cluster.
+When you're ready, apply the `pacrd` manifest to your cluster:
+
+```
+kubectl apply -f pacrd-0.1.0.yaml
+```
+
+Alternatively, if you _really_ trust us, download and install in one command:
+
+```
+kubectl apply -f https://engineering.armory.io/manifests/pacrd-0.1.0.yaml
+```
+
+<!-- 
+
+Since we're still experimental, and we don't allow a lot of configuration right
+now, I'm omitting this section of the docs. When we start to firm up the details
+of how this will be installed and run we can expand the Installation section
+and include details about customizing an install.
+
+# Installation
+
+!-->
+
+# Usage
+
+Once you have PaCRD installed and running in your cluster you can begin to define
+your Applications and Pipelines and apply them to the cluster.
+
+While this product is in [**Experimental**], Kind objects for PaCRD will live
+under the `pacrd.armory.spinnaker.io/v1alpha1` version moniker.
+
+## Applications
+
+Applications are a logical construct in Spinnaker that allow you to group
+resources under a single name. You can read more about [applications in the
+upstream docs][app-docs].
+
+### Current Limitations
+
+The following sections document limitations with various versions of the PaCRD
+controller for this resource.
+
+#### v0.1.x
+
+- Documentation for available Application spec fields must unfortunately be
+found in the installation manifest for this controller. You can do so by
+grepping for `applications.pacrd.armory.spinnaker.io` in the installation
+manifest. Fields are documented under `spec.validation.openAPIV3Schema`.
+
+### Creating
+
+Applications in Kubernetes can be defined with the same options you would define
+in the UI. The following example shows how you might define a simple Application:
+
+```yaml
+# file: application.yaml
+apiVersion: pacrd.armory.spinnaker.io/v1alpha1
+kind: Application
+metadata:
+  name: myapplicationname
+spec:
+  email: sam.tribal@armory.io
+  description: My Application is a catalogue of Widgets surfaced by an API.
+```
+
+*Note: Application names must adhere to both [Kubernetes][kube-name-doc]
+__and__ Spinnaker name standards.*
+
+If you've defined this in an `application.yaml` file then you can create it in
+your cluster by running the following `kubectl` command:
+
+```
+kubectl apply -f application.yaml
+```
+
+Once applied, you can check on the status of your application by using either
+the `get` or `describe` commands. `kubectl` will recognize either `app` or
+`application` for the resource kind:
+
+```sh
+❯ kubectl get app myapplicationname
+NAME                URL                                                             LASTCONFIGURED   STATUS
+myapplicationname   http://spinnaker.io/#/applications/myapplicationname/clusters   7m26s            Created
```

Comment:
I'd use something like  `https://spinnaker.acmecorp.com/....` because spinnaker.io is real and users may wonder if that's what it'll do.